docs/example/vllm/lws.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: leaderworkerset.x-k8s.io/v1
+kind: LeaderWorkerSet
+metadata:
+  name: vllm
+spec:
+  replicas: 2
+  leaderWorkerTemplate:
+    size: 2
+    restartPolicy: RecreateGroupOnPodRestart
+    leaderTemplate:
+      metadata:
+        labels:
+          role: leader
+      spec:
+        containers:
+          - name: vllm-leader
+            image: kube-ai-registry.cn-shanghai.cr.aliyuncs.com/kube-ai/vllm:0.4.1
+            env:
+              - name: RAY_CLUSTER_SIZE
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.annotations['leaderworkerset.sigs.k8s.io/size']
+            command:
+              - sh
+              - -c
+              - "/vllm-workspace/ray_init.sh head --ray_cluster_size=$RAY_CLUSTER_SIZE; 
+                 python3 -m vllm.entrypoints.openai.api_server --port 8080 --model facebook/opt-125m --gpu-memory-utilization 0.95 --tensor-parallel-size 2"
+            resources:
+              limits:
+                nvidia.com/gpu: "1"
+              requests:
+                cpu: "4"
+                memory: 8Gi
+                nvidia.com/gpu: "1"
+            ports:
+              - containerPort: 8080
+            readinessProbe:
+              tcpSocket:
+                port: 8080
+              initialDelaySeconds: 15
+              periodSeconds: 10
+    workerTemplate:
+      spec:
+        containers:
+          - name: vllm-worker
+            image: kube-ai-registry.cn-shanghai.cr.aliyuncs.com/kube-ai/vllm:0.4.1
```

Comment:
Add a comment to explain that this image is build with the Dockerfile under `./build`.

Reply:
done